README.md

```diff
@@ -51,32 +52,37 @@ See the [Roadmap](https://github.com/hyperledger/aries-framework-javascript/issu
 
 ### Prerequisites
 
-Aries Framework JavaScript depends on the indy-sdk which has some manual installation requirements. Before installing dependencies make sure to [install](https://github.com/hyperledger/indy-sdk/#installing-the-sdk) `libindy` and have the right tools installed for the [NodeJS wrapper](https://github.com/hyperledger/indy-sdk/tree/master/wrappers/nodejs#installing). The NodeJS wrapper link also contains some common troubleshooting steps. The [Dockerfile](./Dockerfile) contains everything needed to get started with the framework. See [Usage with Docker](#usage-with-docker) for more information.
+Aries Framework JavaScript depends on the indy-sdk which has some manual installation requirements.
 
-> If you're having trouble running this project, please the the [troubleshooting](./TROUBLESHOOTING.md) section. It contains the most common errors that arise when first installing libindy.
+#### NodeJS
 
-### Installing
+Follow the instructions [here](https://github.com/hyperledger/indy-sdk/#installing-the-sdk) to install `libindy`. Also make sure to have the right tools installed for the [NodeJS wrapper](https://github.com/hyperledger/indy-sdk/tree/master/wrappers/nodejs#installing). The NodeJS wrapper link also contains some common troubleshooting steps.
+
+If you don't want to install the dependencies yourself, the [Dockerfile](./Dockerfile) contains everything needed to get started with the framework. See [Usage with Docker](#usage-with-docker) for more information.
+
+> If you're having trouble running this project, please the the [troubleshooting](./TROUBLESHOOTING.md) section. It contains the most common errors that arise when first installing libindy.
```

Comment:
Double "the" in "please the the [troubleshooting]"

Reply:
Updated!